### PR TITLE
Feature/event page

### PR DESCRIPTION
**Pull request by ldanet**

### CODE REVIEW

File: ietf/events/templates/events/event_page.html [hunk 1]
```diff
@@ -76,44 +60,44 @@ <h4>{{ link_group.value.title }}</h4><span class="icon ion-ios-arrow-down"></spa
 
 
                 {% if self.venue %}
-                    <div class="body body-panel body-panel-rule bleed-left clearfix">
+                    <div class="border-bottom border-primary p-3">
                         {% if self.venue_section_title %}
-                            <h2 class="section-title">{{ self.venue_section_title }}</h2>
+                            <h2  class="h4">{{ self.venue_section_title }}</h2>
```
Comment (liamjohnston) [inline ietf/events/templates/events/event_page.html:65]: extra space 🔍

File: ietf/events/templates/events/event_page.html [hunk 2]
```diff
@@ -7,65 +7,49 @@
     <div class="no-overflow-x">
 
         {% image self.main_image max-4000x4000 as bg_image %}
-        <div class="header-image-full rule-desktop" style="background-image: url({{ bg_image.url }});">
-            <div class="container">
-                {% include 'includes/breadcrumbs.html' %}
+        <div class="event_page__jumbotron rule-desktop" style="background-image: url({{ bg_image.url }});">
```
Comment (liamjohnston) [inline ietf/events/templates/events/event_page.html:10]: is `rule-desktop` a legacy style?

**Review by liamjohnston — APPROVED** [review]

File: ietf/events/templates/events/event_page.html (hunk 2 above)
Comment (ldanet) [reply ietf/events/templates/events/event_page.html:10]: Indeed!

File: ietf/events/templates/events/event_page.html (hunk 1 above)
Comment (ldanet) [inline ietf/events/templates/events/event_page.html:65]:
```suggestion
                            <h2 class="h4">{{ self.venue_section_title }}</h2>
```

File: ietf/events/templates/events/event_page.html (hunk 2 above)
Comment (ldanet) [inline ietf/events/templates/events/event_page.html:10]:
```suggestion
        <div class="event_page__jumbotron" style="background-image: url({{ bg_image.url }});">
```

**Pull request merged**